src/PostgREST/Types.hs

```diff
@@ -1,70 +1,84 @@
 module PostgREST.Types where
-import Data.Text
-import Data.Tree
+
+import           Data.Text (Text)
+import           Data.List (find)
+import           Data.Tree
 import qualified Data.ByteString.Char8 as BS
-import Data.Aeson
-import Data.Map
+import           Data.Aeson
+import           Data.Map (Map)
 
 data DbStructure = DbStructure {
-  columns :: [Column]
-, relations :: [Relation]
-, primaryKeys :: [PrimaryKey]
-}
+  dbTables :: [Table]
+, dbColumns :: [Column]
+, dbRelations :: [Relation]
+, dbPrimaryKeys :: [PrimaryKey]
+} deriving (Show, Eq)
+
+dbFindTable :: DbStructure -> Schema -> Text -> Maybe Table
```

Comment:
My logic was that this module exports functions like `dbTables` so functions like these could go here to. You're right though, they should probably be in DbStructure.hs